modular_skyrat/modules/wargame_projectors/code/game_kit.dm

```diff
@@ -15,5 +15,429 @@
+								17 - 20, The ship suffers catastrophic structural damage, and it will be lost if this compounds with another problem <br>
+							<br>
+							If a ship is lost in combat, then one of two things will happen. If the reactor was damaged
+							at the time of the vessel being lost, then it should catastrophically fail and cause
+							any ships near it should take damage from the detonation, and the ship itself should be
+							removed from the board. If the reactor was not damaged when the vessel was lost however,
+							then the ship's color should be changed to a neutral grey, where it will remain as an empty
+							hulk that can be used for cover or concealment by other ships.
+						<br>
+						<b>Objectives and Victory</b> <br>
+							At the start of the match, the game master should assign each team, or player, a specific objective
+							that they must complete in order to win. This objective can be something simple like complete
+							destruction of the enemy forces. However more complex objectives, like escorting a civilian
+							ship to a nearby planet, protecting a station of attack, and so on, are all possible. In the end
+							it is up to the game master to come up with a scanario and objectives.
+							<br>
+							When the game master determines these objectives to be complete, or that some kind of game
+							ending condition has been achieved, then they may end the game at will.
+						<br>
+						*Wargame: Blue Lizard is a copyright protected title under ownership of NanoTrasen
+				</body>
+				</html>
+			"}
+
+/obj/item/book/manual/wargame_rules/examples
+	name = "Wargame: Blue Lizard - Example Ships and Scenarios"
+	icon_state ="book1"
+	starting_author = "NanoTrasen Naval Wargames Commission"
+	starting_title = "Wargame: Blue Lizard - Example Ships and Scenarios"
+	starting_content = {"
+				<head>
+				<meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
+				<style>
+				h1 {font-size: 18px; margin: 15px 0px 5px;}
+				h2 {font-size: 15px; margin: 15px 0px 5px;}
+				li {margin: 2px 0px 2px 15px;}
+				ul {list-style: none; margin: 5px; padding: 0px;}
+				ol {margin: 5px; padding: 0px 15px;}
+				</style>
+				</head>
+				<body>
+				<center>
+				<b style='font-size: 12px;'>Official publication of the NanoTrasen Naval Wargames Commission</b>
+				<h1>Wargame: Blue Lizard</h1>
+				</center>
+				<h2>Introduction</h2>
+					Should you not have read the example ruleset that should have come with this book, then it is
+					highly suggest that you read through that first. This publication will contain a list of example
+					ships, and scenarios for using in games. It should be noted that it is not required to use the
+					ships in this book, and that making your own with your game master is encouraged.
+				<br>
+				<h2>Example Ships</h2>
+					<b>NanoTrasen</b> <br>
```

Comment:
Nanotrasen not NanoTrasen.

Reply:
NanoTrasen is the correct spelling

Reply:
> NanoTrasen is the correct spelling

It's written Nanotrasen in the lore doc tho.
Whatever tbqh.